### PR TITLE
UN-29 Create schema for users, organization and events

### PR DESCRIPTION
**Pull request by om-ganesh**
This is the first draft of the same database which we discussed on last weekend. I just made a PR to collect the reviews.

### CODE REVIEW

**Comment by om-ganesh** [comment]
> Could we change the README.doc to a markdown format so that it can be rendered on github

This document has just a single image (to share the db schema). I will be uploading the detail schema design description in the readme.md file.

File: packages/database/CreateSchema.sql [hunk 1]
```diff
@@ -0,0 +1,234 @@
+--CREATE DATABASE UnitedNepaliDb; 
+	
+--Create a developer log table
+CREATE TABLE DevLog(
+	Id int  NOT NULL,
```
[the remark this answers is not in the document]
Comment (sonalranjit) [reply packages/database/CreateSchema.sql]: Sorry i thought I submitted this review already

[the remark this answers is not in the document]
Comment (om-ganesh) [reply packages/database/CreateSchema.sql]: Sure will follow in just next commit. Please correct me, if wrong, the conventions to be followed is: **key words in upper case and names in lower case, separated by underscore if two+ words**

[the remark this answers is not in the document]
Comment (sonalranjit) [reply packages/database/CreateSchema.sql]: yes exactly, names in *lowercase, separated by undersocre if two + words*

[the remark this answers is not in the document]
Comment (om-ganesh) [reply packages/database/CreateSchema.sql]: Addressed

File: services/db/create_schema.sql [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+	));
+
+--Security profiles 
+CREATE TABLE security_profile(
+	id VARCHAR(128) NOT NULL,
+	created_date TIMESTAMP NOT NULL,
+	description VARCHAR(256) NULL,
+	CONSTRAINT PK_SecurityProfile PRIMARY KEY  
+	(
+		Id
```
[the remark this answers is not in the document]
Comment (om-ganesh) [reply services/db/create_schema.sql]: good catch

File: services/db/create_schema.sql [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+	(
+	id
+	));
+
+--Security profiles 
+CREATE TABLE security_profile(
+	id VARCHAR(128) NOT NULL,
+	created_date TIMESTAMP NOT NULL,
+	description VARCHAR(256) NULL,
+	CONSTRAINT PK_SecurityProfile PRIMARY KEY
```
[the remark this answers is not in the document]
Comment (om-ganesh) [reply services/db/create_schema.sql]: good catch. done

**Pull request merged**